src/main/java/org/zalando/nakadi/controller/EventPublishingController.java

```diff
@@ -62,7 +68,13 @@ public ResponseEntity postEvent(@PathVariable final String eventTypeName,
         }
     }
 
-    private ResponseEntity postEventInternal(final String eventTypeName,
+    @RequestMapping(value = "/event-types/{eventTypeName}/events", method = HEAD)
+    public ResponseEntity<?> headEvent(@PathVariable final String eventTypeName, final Client client) {
```

Comment:
I think this endpoint is questionable.
It was not reviewed by the guild and I think it doesn't align to API guidelines: http://zalando.github.io/restful-api-guidelines/http/Http.html#head